hpx/lcos/channel.hpp

```diff
@@ -457,8 +455,7 @@ namespace hpx { namespace lcos
                 send_channel<T>, lcos::server::channel<T>
             > base_type;
 
-        HPX_CONSTEXPR_OR_CONST
-        static std::size_t default_generation = std::size_t(-1);
+        constexpr static std::size_t default_generation = std::size_t(-1);
```

Comment:
Should we write `constexpr static` or `static constexpr`? I don't care, but it should be consistent.